dev-packages/node-integration-tests/suites/public-api/LocalVariables/local-variables-out-of-app-default.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-unused-vars */
+
+const Sentry = require('@sentry/node');
+// const { loggingTransport } = require('@sentry-internal/node-integration-tests'); is throwing error that package not found, so using relative path
+const { loggingTransport } = require('../../../src/index.ts');
+
+// make sure to create the following file with the following content:
+// function out_of_app_function() {
+//   const outOfAppVar = 'out of app value';
+//   throw new Error('out-of-app error');
+// }
+
+// module.exports = { out_of_app_function };
```

Comment:
This is far too brittle, but not sure how to best do this. Maybe @timfish you have some ideas?

Reply:
The easy answer is probably just to create the file in a `beforeAll` handler in the test, then delete it in an `afterAll` handler.